auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -27,16 +28,18 @@ const int _kMergeCountPerRepo = 1;
 ///
 /// For pull requests where an 'autosubmit' label was added in pubsub,
 /// check if the pull request is mergable.
-class CheckPullRequest extends RequestHandler {
+class CheckPullRequest extends ApiRequestHandler {
```

Comment:
Instead of `ApiRequestHandler` make this `AuthenticatedRequestHandler`

Reply:
Changed the file and class name.